src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/TvpTest.cs

```diff
@@ -374,8 +371,10 @@ private void TestTVPPermutations(SteStructuredTypeBoundaries bounds, bool runOnl
                 // Set up base command
                 SqlCommand cmd;
                 SqlParameter param;
-                cmd = new SqlCommand(GetProcName(tvpPerm));
-                cmd.CommandType = CommandType.StoredProcedure;
+                cmd = new SqlCommand(GetProcName(tvpPerm))
+                {
+                    CommandType = CommandType.StoredProcedure
+                };
```

Comment:
I don't see any benefit on some changes like this in the tests. It's only added more lines.